src/cmd/setup-tachyon.js

```diff
@@ -407,6 +407,16 @@ Welcome to the Particle Tachyon setup! This interactive command:
 		return manager.download({ url, outputFileName, expectedChecksum, options: { alwaysCleanCache } });
 	}
 
+	_isProbablyAPath(input) {
```

Comment:
Can you do fs.exists for this?

Reply:
If that's not easy to tweak, this is good enough for now. Just note to change it later.

Reply:
yeah. I want to implement some permissions on this one. since fs.exist could return false because of permissions or the file doesn't exist. 
I've created this one just to start checking if the name could be a "filename" and go further the next steps where we validate if the path exists

Reply:
I've changed here https://github.com/particle-iot/particle-cli/pull/805/commits/15983c018c28e5a354e24b34438ae976f6a25a05. 
I'll revisit this one when I work on permissions issue.